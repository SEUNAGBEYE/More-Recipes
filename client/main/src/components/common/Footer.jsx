import React from 'react';
import PropTypes from 'prop-types';

/**
 * @description - Renders Footer Component
 *
 * @param {Object} props
 *
 * @returns {Jsx} Jsx
 */
const Footer = (props) => (
  <div>
    <footer style={props.style}>
      <div>
        <strong>&copy; Recipes</strong>
      </div>
    </footer>
  </div>
);

const propTypes = {
  style: PropTypes.object
};

Footer.propTypes = propTypes;

export default Footer;

