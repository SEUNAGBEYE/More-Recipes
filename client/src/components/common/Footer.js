import React from 'react';

const Footer = () => {
  return (
    <div>
      <footer>
        <div>
          <strong>&copy; Recipes</strong>
        </div>
      </footer>
    </div>
  );
};

export default Footer