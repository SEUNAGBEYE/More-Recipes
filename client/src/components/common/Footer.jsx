import React from 'react';

const Footer = (props) => {
  return (
    <div>
      <footer style={props.style}>
        <div>
          <strong>&copy; Recipes</strong>
        </div>
      </footer>
    </div>
  );
};

export default Footer