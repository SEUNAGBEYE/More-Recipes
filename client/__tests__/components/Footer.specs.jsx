
import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../main/src/components/common/Footer';

describe('# Footer', () => {
  const wrapper = shallow(<Footer/>);
  it('should render successfully', (done) => {
    expect(wrapper).toBeDefined();
    expect(wrapper.getElement().type).toBe('div');
    expect(wrapper).toMatchSnapshot()
    done();
  });
});
