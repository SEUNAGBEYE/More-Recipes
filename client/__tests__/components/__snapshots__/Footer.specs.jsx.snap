// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`# Footer should render successfully 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <footer
        style={undefined}
      >
        <div>
          <strong>
            © Recipes
          </strong>
        </div>
      </footer>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div>
          <strong>
            © Recipes
          </strong>
        </div>,
        "style": undefined,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <strong>
            © Recipes
          </strong>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "© Recipes",
          },
          "ref": null,
          "rendered": "© Recipes",
          "type": "strong",
        },
        "type": "div",
      },
      "type": "footer",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <footer
          style={undefined}
        >
          <div>
            <strong>
              © Recipes
            </strong>
          </div>
        </footer>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div>
            <strong>
              © Recipes
            </strong>
          </div>,
          "style": undefined,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <strong>
              © Recipes
            </strong>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "© Recipes",
            },
            "ref": null,
            "rendered": "© Recipes",
            "type": "strong",
          },
          "type": "div",
        },
        "type": "footer",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
